apps/financial-aid/backend/src/environments/environment.ts

```diff
@@ -32,6 +32,7 @@ const prodConfig = {
     },
   },
   baseUrl: process.env.BASE_URL,
+  veitaUrl: process.env.VEITA_BASE_URL,
```

Comment:
baseUrl and veitaUrl, do we have better or more clear names 😄

Reply:
Sure, any suggestions? 😄